cypress/e2e/LoginProxy.cy.ts

```diff
@@ -0,0 +1,13 @@
+describe("Input Login tests", () => {
+    beforeEach(() => {
+      cy.viewport(1920, 1080);
+    });
+  
+    it("Should submit the form with valid data", () => {
```

Comment:
Where are you registering this user? 

Since the register feature haven not been implemented yet, you can remove this tests file and implement the login tests in the Pull Request to implemente the signup.